_schemaV2.graphql

```diff
@@ -4764,6 +4764,9 @@ input BulkUpdateArtworksMetadataInput {
   # The artwork condition to be assigned
   conditionDescription: String
 
+  # Array of dates as numbers to be assigned
+  dates: [Int]
```

Comment:
@lidimayra, an array of numbers and not strings is what we need here, right (Gravity changes: https://github.com/artsy/gravity/pull/19283)?